src/gui/accountmanager.cpp

```diff
@@ -750,7 +768,7 @@ void AccountManager::deleteAccount(OCC::AccountState *account)
 void AccountManager::updateServerHasValidSubscriptionConfig()
 {
     auto serverHasValidSubscription = false;
-    for (const auto &account : std::as_const(_accounts)) {
+    for (const auto &account : _accounts) {
```

Comment:
please revert
this is generating a warning with clazy (https://github.com/KDE/clazy/blob/master/docs/checks/README-range-loop-detach.md)